homework/index.js

```diff
@@ -28,14 +28,75 @@
     });
     return elem;
   }
+  // convert date time
+  function convertDataTime(dateTimeTxt) {
+    const dateTime = new Date(dateTimeTxt);
+    return dateTime.toLocaleString();
+  }
+  // sort repositories by name
+  function sortElements(a, b) {
+    return a.name.localeCompare(b.name);
+  }
 
   function renderRepoDetails(repo, ul) {
-    createAndAppend('li', ul, { text: repo.name });
+    const repoList = createAndAppend('li', ul, { class: 'rep-list' });
```

Comment:
You have two other CSS class names, viz. `repo-header` and `repo-details`. For consistency, it is best to use `repo-list` here.